bundle/jsky.app.ot/src/main/java/jsky/app/ot/too/TooAlertPanel.java

```diff
@@ -12,17 +8,16 @@
 
 import javax.swing.*;
 import java.awt.*;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
 import java.util.TimeZone;
-import java.util.Date;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
 
 /**
  * A panel that displays a TOO alert according to the spec in SCT-233.
  */
 public final class TooAlertPanel extends JPanel {
     private static final String TIME_ONLY_PATTERN = "HH:mm:ss";
-    private static final String TIME_DATE_PATTERN = "yyyy/MM/dd HH:mm:ss";
+    private static final String TIME_DATE_PATTERN = "YYYY-MMM-dd HH:mm:ss";
```

Comment:
We should probably follow up on this after the release to extract this pattern in the OT so we don't have to repeat it in a lot of places.

Reply:
Absolutely. This is repeated over and over again in so many places. `DateUtil` should be reworked, I'm thinking, to have `DateTimeFormatter`s so that we don't have to declare these over and over again.

I'll make a task for this so we don't forget it.